README.md

```diff
@@ -340,6 +341,28 @@ func isError(error: Error?) -> Bool {
 }
 ```
 
+### Implicit Getters ###
+
+When overriding only the getter of a property, omit the use of `get`:
```

Comment:
The `set` case will be compiler enforced. You _have_ to specify a `set` in order to override it -- I don't think it's worth mentioning here as a result. Because we're basically saying "In other cases, make sure that your code compiles"